pallets/rad-claims/Cargo.toml

```diff
@@ -1,33 +1,31 @@
 [package]
 name = 'pallet-rad-claims'
```

Comment:
I think w can call them claims instead. Dont want to carry RAD over to new chain :) and once AIR comes in, CFG would be confusing